hazelcast/src/main/java/com/hazelcast/internal/management/request/GetCacheEntryRequest.java

```diff
@@ -193,6 +201,7 @@ public void writeData(ObjectDataOutput out) throws IOException {
             out.writeLong(creationTime);
             out.writeLong(lastAccessTime);
             out.writeLong(accessHit);
+            out.writeObject(expiryPolicy);
```

Comment:
can this cause a compatibility problem?

Reply:
I'm copying the message I've sent over Slack to here as Slack seems to be down:

@mustafaiman  I don't see any possible compatibility problems with Management Center. The important methods for MC compatibility are `GetCacheEntryRequest#writeResponse` and `GetCacheEntryRequest#fromJson`. I think what @ahmetmircik  meant was more about compatibility between members. From what I know about rolling upgrade, seems like it might cause compatibility problems in a mixed version cluster but @vbekiaris should know better.

Reply:
I thought this is a message that goes between management center and the member. I do not think this is used between member to member communication. If it is not used between member to member communication, it should be fine. @ahmetmircik @emre-aydin

Reply:
Yes it is but this PR contains no changes to the parts relevant for MC communication. Upon receiving this message, member executes an entry processor (possibly on another member) to get an entry and sends back the result to MC.